OpenEdXMobile/res/layout/fragment_dashboard_error_layout.xml

```diff
@@ -1,27 +1,14 @@
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We can delete this file and use the parent layout for errorLayout defined in `fragment_course_tabs_dashboard.xml`